lib/oxidized/model/edgecos.rb

```diff
@@ -44,6 +50,7 @@ class EdgeCOS < Oxidized::Model
 
   cfg :telnet, :ssh do
     post_login 'terminal length 0'
+    post_login 'terminal width 300'
```

Comment:
Is 300 the maximum?

Reply:
Yes. 300 is the maximum. At least on the devices (software versions) that I have at hand it is impossible to set terminal width to 0, accepted range is 80 - 300.